.github/workflows/build-and-publish.yml

```diff
@@ -0,0 +1,72 @@
+# This is a basic workflow to help you get started with Actions
+name: Bundle building and pushing
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the main branch
+on: [workflow_dispatch]
+
+env:
+  GO_VERSION: '1.16.1'
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: ${{ env.GO_VERSION }}
+      - name: Linter
+        uses: golangci/golangci-lint-action@v2
+        with:
+          version: latest
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: ${{ env.GO_VERSION }}
+      - name: Download modules
+        run: go mod download
+      - name: Build project
+        run: go build ./cmd/aks-periscope
+      - name: Run the changelog reader
+        id: changelog_reader
+        uses: ./
+      - name: Display output
+        run: |
+          echo "Version: ${{ steps.changelog_reader.outputs.version }}"
+      - name: Create a Release
+        id: create_release
+        uses: actions/create-release@v1
+        env:
+            GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }} 
+        with:
+          tag_name : ${{ steps.changelog_reader.outputs.current-version}}
+          release_name: ${{ steps.changelog_reader.outputs.current-version}}
+          body: Publish ${{ steps.changelog_reader.outputs.current-version}}
+      # Lowercase my github ownername.
+      - name: Set Environment Variables
+        run: |
+          OWNER="$(echo "${{ github.repository_owner }}" | tr '[:upper:]' '[:lower:]')"
```

Comment:
Where is `repository_owner` property set in a given repo? 

Is this something we instead want to allow for multiple people, such as everyone under a certain team (ie. https://github.com/orgs/Azure/teams/agentbakerwrite/members)?

Reply:
Indeed, `repository_owner ` this is default set at github level. Thank you for asking.

* For more details this is a **Good** read: https://docs.github.com/en/actions/learn-github-actions/environment-variables 

* I would say as long as this list is safe and actual contributor to the project then there is no hard, we trust these folks, but if that is not the case, best is to remove those aliases from the list of access.

* Idea is no - I would say - No one will go and triggers release for fun :) especially it is bound to the `ChangeLog`